### ISSUE TITLE
[impovement]: better output for { depth: 0 } option

### ISSUE THREAD

**Issue by everdimension** [ljharb/qs#246]
Desired behavior:

```js
qs.parse('items[a]=one&items[b]=two', { depth: 0 });
// —> { 'items[a]': 'one', 'items[b]': 'two' }
```

Current behavior:

```js
qs.parse('items[a]=one&items[b]=two', { depth: 0 });
// –> { items: { '[a]': 'one', '[b]': 'two' } }
```

It seems to me that the current output is quite useless, but having a one-level-deep can be quite handy, for example, when dealing with form inputs.

**Comment by ljharb** [comment]
Hmm, there's no explicit tests for a depth value of zero.

To be honest, both of those look pretty useless to me; but I agree with you that the former is what I'd expect with a depth of zero. What do you get with `depth: 1`?

**Comment by everdimension** [comment]
with `{ depth: 1 }` you get `{ items: { a: 'one', b: 'two' } }` as expected.

Would you accept a PR for `depth: 0`?

**Comment by ljharb** [comment]
I think a PR with associated tests would be great.

**Comment by ljharb** [comment]
This should be fixed by #326.
